[ingress-nginx] Add maxmind download alerts (#14889)
deckhouse/deckhouse: modules/402-ingress-nginx/images/controller-1-12/patches/README.md

### 014-verbose-maxmind-logs.patch

Added additional logging for downloading GeoIP databases from the MaxMind service.

### 015-maxmind-alerts.patch

> The metric `geo2_download_errors_total` is added, indicating the number of errors when downloading geo databases from the MaxMind service.

Reviewer: ```suggestion
The metric `geoip_errors_total` has been added, which indicates the number of errors related to GeoIP, specifically download errors (`type="download"`).
```
